addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -55,6 +55,22 @@ const MassMailingWysiwyg = Wysiwyg.extend({
                 }
             }
         }
+        const signCommand = commands.find(command => command.title === 'Signature');
+        if (signCommand) {
+            signCommand.callback = () => {
+                this._rpc({route: '/web/session/get_session_info'}).then(session => (
```

Comment:
out of curiosity, do you need to call `/web/session/get_session_info` to know the current user ID? I thought it was available via `getSession().uid`, `userService.userId` and whatnot. Am I missing something?

Reply:
You're not missing anything, _I_ was ;-) I corrected it to use `getSession()`, thanks.